src/auslib/blobs/base.py

```diff
@@ -135,6 +136,14 @@ def merge_dicts(ancestor, left, right):
 class Blob(dict):
     jsonschema = None
 
+    class ServeUpdate(enum.Enum):
```

Comment:
suggestion (non-blocking): Move this to the top level of the file. I mainly suggest this because things like `candidate == blob.ServeUpdate.Maybe` further up comparing a return value from a blob method to a fixed property of a blob, which just seems a bit odd. `candidate == ServeUpdate.Maybe` is a bit more intuitive to me. Totally up to you though.

Reply:
Done.